package.json

```diff
@@ -20,9 +20,14 @@
   },
   "devDependencies": {
     "@nomiclabs/hardhat-ethers": "^2.0.4",
+    "@nomiclabs/hardhat-etherscan": "^3.0.0",
     "@nomiclabs/hardhat-waffle": "^2.0.1",
+    "@openzeppelin/contracts": "^4.4.2",
+    "@openzeppelin/contracts-upgradeable": "^4.4.2",
+    "@openzeppelin/hardhat-upgrades": "^1.13.0",
     "@openzeppelin/test-helpers": "^0.5.15",
     "chai": "^4.3.4",
+    "dotenv": "^12.0.4",
```

Comment:
What's the purpose of the dotenv? could we remove this and keep this project agnostic to the user's dev environment?